src/core_modules/capture-core/components/WidgetTrackedEntityRelationship/NewTrackedEntityRelationship/NewTrackedEntityRelationship.component.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { Button, IconSearch16, IconAdd16, spacers } from '@dhis2/ui';
+import { withStyles } from '@material-ui/core';
+import { NewTEIRelationshipStatuses } from '../WidgetTrackedEntityRelationship.const';
+import { RelationshipTypeSelector } from './RelationshipTypeSelector/RelationshipTypeSelector';
+
+const styles = {
```

Comment:
Hey, 
Can you have a look at the `WidgetTrackedEntityRelationship` width? On a laptop screen, you can see other widgets below. It looks fine on bigger screens. 

<img width="1081" alt="Screenshot 2022-02-07 at 15 45 18" src="https://user-images.githubusercontent.com/6817073/152810860-b0ff653f-9379-407b-bfb1-8c6aef512103.png">

Thanks!